.github/workflows/instrumented-tests.sh

```
@@ -0,0 +1,29 @@
+
+TARGET=""
+if [ $API_LEVEL -ge 28 ]; then
+    TARGET="google_apis_playstore"
+else
+    TARGET="default"
+fi
+
+set -x
+set +e
+echo "Creating emulator..."
+echo no | $ANDROID_HOME/cmdline-tools/latest/bin/avdmanager create avd --force -n test --package "system-images;android-$API_LEVEL;$TARGET;x86_64"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)